app/routes/form.$version.docs.framework.$framework.$.tsx

```diff
@@ -1,20 +1,57 @@
 import type { LoaderFunctionArgs, MetaFunction } from '@remix-run/node'
+import type { GithubDocsConfig } from '~/routes/form'
 import { repo, getBranch } from '~/routes/form'
 import { DefaultErrorBoundary } from '~/components/DefaultErrorBoundary'
 import { seo } from '~/utils/seo'
 import { useLoaderData, useParams } from '@remix-run/react'
 import { Doc } from '~/components/Doc'
 import { loadDocs } from '~/utils/docs'
+import { fetchRepoFile } from '~/utils/documents.server'
 
 export const loader = async (context: LoaderFunctionArgs) => {
```

Comment:
The body of the `loader` function is basically identical in this file and [`app/routes/store.$version.docs.framework.$framework.$.tsx`](https://github.com/TanStack/tanstack.com/pull/133/files#diff-cec2d983f6a6adc45d30d2ffe186f539471e0d67e4948e2c3e2b8b5ed15276caR11-R55). If we want to follow the DRY practices, we can extract the shared logic to a helper function.